### ISSUE TITLE
Cannot start training from csresnext50-panet-spp-original-optimal.cfg?

### ISSUE THREAD

**Issue by eric-tc** [AlexeyAB/darknet#4967]
Hi,

i'm trying to train csresnext50 starting from csresnext50-panet-spp-original-optimal.cfg. I use this command `./darknet detector train cfg/coco.data customConfig/csresnext50-panet-spp-original-optimal.cfg csresnext50-omega_final.weights -dont_show -mjpeg_port 8090 -map`

i want to train it on coco dataset but only with small number of classes. I have modified the cfg according to guidelines. 

The problem is that i cannot start training. What i get is `

...
29 route  128 92 	                           ->   19 x  19 x1024 
 130 conv    512       1 x 1/ 1     19 x  19 x1024 ->   19 x  19 x 512 0.379 BF
 131 conv   1024       3 x 3/ 1     19 x  19 x 512 ->   19 x  19 x1024 3.407 BF
 132 conv    512       1 x 1/ 1     19 x  19 x1024 ->   19 x  19 x 512 0.379 BF
 133 conv   1024       3 x 3/ 1     19 x  19 x 512 ->   19 x  19 x1024 3.407 BF
 134 conv    512       1 x 1/ 1     19 x  19 x1024 ->   19 x  19 x 512 0.379 BF
 135 conv   1024       3 x 3/ 1     19 x  19 x 512 ->   19 x  19 x1024 3.407 BF
 136 conv     57       1 x 1/ 1     19 x  19 x1024 ->   19 x  19 x  57 0.042 BF
 137 yolo
[yolo] params: iou loss: ciou (4), iou_norm: 0.07, cls_norm: 1.00, scale_x_y: 1.05
nms_kind: greedynms (1), beta = 0.600000 
Unused field: 'uc_normalizer = 0.07'
Unused field: 'beta1 = 0.6'
Total BFLOPS 99.581 
avg_outputs = 1105209 
 Allocate additional workspace_size = 52.43 MB 
Loading weights from csresnext50-omega_final.weights...
 seen 64, trained: 153600 K-images (2400 Kilo-batches_64) 
Done! Loaded 82 layers from weights-file 
Learning Rate: 0.00261, Momentum: 0.949, Decay: 0.0005
Saving weights to /home/vision-e/Desktop/2020/darknet/backup//csresnext50-panet-spp-original-optimal_final.weights
`

Why training is not starting?

Thanks

**Comment by WongKinYiu** [comment]
you need `-clear`

**Comment by eric-tc** [comment]
Hi @WongKinYiu ,

thanks for the answer. Where i can find the backbone of `csresnext50-panet-spp-original-optimal.cfg` ?

**Comment by WongKinYiu** [comment]
you can use `partial` function to get the first 80 layers of `imagenet` pre-trained model or first 112 layers of `coco` pre-trained model.

* imagenet pre-trained 1: [csresnext50.weights](https://drive.google.com/file/d/1IxKu5lAYCo4UpzAl5pOBIDAvC843vjn1/view)

* imagenet pre-trained 2: [csresnext50-gamma_final.weights](https://drive.google.com/file/d/1pm_zwf4e0kuShiRz9AW2GJV22Jjc1xjW/view)

* coco pre-trained: [csresnext50-panet-spp-original-optimal_final.weights](https://drive.google.com/file/d/1CP1IGmo29TCVr6SxozRE16yqG67JXr2b/view)

**Comment by eric-tc** [comment]
ok Thanks

**Comment by isra60** [comment]
@WongKinYiu  Hi. The imagenet pretrained weights you share are for image classification instead of object detection??? And if this is true do yo know where is the cfg file for classification?

**Comment by WongKinYiu** [comment]
The first 80 layers are same, so just use cfg of object detection to get pre-trained weights is ok.

the cfg file of CSPResNeXt50 image classification model is [here](https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/master/cfg/csresnext50.cfg).
